_build/partials/header.hbs

```diff
@@ -38,6 +38,11 @@
   <script>
     var rootUrl = "{{rootUrl}}";
   </script>
+
+  <!-- Fathom - beautiful, simple website analytics -->
+  <script src="https://cdn.usefathom.com/script.js" data-site="UENARRJI" defer></script>
```

Comment:
Let's also still add `data-excluded-domains="http://devext-fe.sinergise.com/,localhost"` to not track our internal testing.

*There is no local deployment that would use the fathom script right? If yes, we don't need to include "localhost".

Reply:
Pls. still remove the localhost as it doesn't do anything here.

Reply:
Sorry, I didn't tell you correctly - there is local deployment (running the project locally when developing). 

Tested it without the `localhost` when running the project locally and the usage gets reported, so I would leave this in.

Reply:
No problem! We should add that info at some point to the ReadMe than.

Reply:
Currently it's described (only) in the `CONTRIBUTING.md`, which is easily missed, so it would probably be enough to link it in the `README.md`